connect/tests/src/test/java/com/telenor/connect/utils/IdTokenValidatorTest.java

```diff
@@ -119,15 +131,17 @@ public void authorizedPartyNotEqualClientThrows() throws Exception {
         BDDMockito.given(ConnectSdk.getExpectedIssuer())
                 .willReturn("https://connect.telenordigital.com/oauth");
 
-        JWTClaimsSet claimsSet = new JWTClaimsSet();
-        claimsSet.setIssuer("https://connect.telenordigital.com/oauth");
-        claimsSet.setAudience("connect-tests");
-        claimsSet.setExpirationTime(oneHourIntoFuture);
-        claimsSet.setIssueTime(now);
-        claimsSet.setCustomClaim("azp", "NOT connect-tests");
-
-        SignedJWT signedJWT = new SignedJWT(new JWSHeader(JWSAlgorithm.ES256), claimsSet);
-        signedJWT.sign(new ECDSASigner(new BigInteger("123")));
+        JWTClaimsSet claimsSet = new JWTClaimsSet.Builder()
+                .issuer("https://connect.telenordigital.com/oauth")
+                .audience("connect-tests")
+                .expirationTime(oneHourIntoFuture)
+                .issueTime(now)
+                .claim("azp", "NOT connect-tests")
+                .build();
+
+        SignedJWT signedJWT = new SignedJWT(new JWSHeader(JWSAlgorithm.HS256), claimsSet);
+        JWSSigner signer = new MACSigner("secret01234567890ABCDEFGHIJKLMNO");
```

Comment:
Extract this into a constant and name it in such a way that it is clear to everyone that this is test data.

Reply:
Same goes for other occurances.

Reply:
@paulrene fixed